src/graphics/device.js

```diff
@@ -901,6 +902,58 @@ pc.extend(pc, function () {
 
         /**
          * @function
+         * @name pc.GraphicsDevice#copyRenderTarget
+         * @description Copies color and/or depth contents of one render target to another. Formats, sizes and anti-aliasing samples must match.
+         * Depth buffer can only be copied on WebGL 2.0.
+         * @param {pc.RenderTarget} source Source render target to copy from. Can be null for back buffer.
+         * @param {pc.RenderTarget} dest Destination render target to copy to. Can be null for back buffer.
+         * @param {Boolean} color Copy color buffer
+         * @param {Boolean} depth Copy depth buffer
+         */
+        copyRenderTarget: function (source, dest, color, depth) {
+            var gl = this.gl;
+            var shaderCopy = false;
+
+            // #ifdef DEBUG
+            if (!this.webgl2 && depth) {
+                console.error("Depth is not copyable on WebGL 1.0");
+            }
+            // #endif
+
+            if (this.webgl2) {
+                if (dest) {
+                    var prevRt = this.renderTarget;
+                    this.renderTarget = dest;
+                    this.updateBegin();
+                    gl.bindFramebuffer(gl.READ_FRAMEBUFFER, source ? source._glFrameBuffer : null);
+                    gl.bindFramebuffer(gl.DRAW_FRAMEBUFFER, dest._glFrameBuffer);
+                    var w = source ? source.width : dest.width;
+                    var h = source ? source.height : dest.height;
+                    gl.blitFramebuffer( 0, 0, w, h,
+                                        0, 0, w, h,
+                                    (color ? gl.COLOR_BUFFER_BIT : 0) | (depth ? gl.DEPTH_BUFFER_BIT : 0),
+                                    gl.NEAREST);
+                    this.renderTarget = prevRt;
+                    gl.bindFramebuffer(gl.FRAMEBUFFER, prevRt ? prevRt._glFrameBuffer : null);
+                } else {
+                    shaderCopy = true;
+                }
+            } else {
+                shaderCopy = true;
+            }
+
+            if (shaderCopy) {
+                if (!this._copyShader) {
+                    var chunks = pc.shaderChunks;
+                    this._copyShader = chunks.createShaderFromCode(this, chunks.fullscreenQuadVS, chunks.outputTex2DPS, "outputTex2D");
+                }
+                this.constantTexSource.setValue(source._colorBuffer);
```

Comment:
It will lead to exception, which will originate from engine thus will look like engine bug.